container-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CMD=${1-docker}
+
+$CMD build -t crc-build .
```

Comment:
docker build can return the id of the image so you don't need to name it and rmi after.